test/hf_test.py

```diff
@@ -917,8 +924,14 @@ def test_case(self):
             ]:
                 self.skipTest("Need to support CLIPVisionModelWithProjection")
 
+            # TODO: support SwinBackbone
```

Comment:
nit: I would refactor this check into a function.

Also seems like we have to backbones to support. what is the missing feature?